src/System/IO/Streams/Handle.hs

```diff
@@ -71,6 +72,27 @@ handleToOutputStream h = makeOutputStream f
 
 
 ------------------------------------------------------------------------------
+-- | Converts a readable and writable handle into an 'InputStream'/'OutputStream'
+-- of strict 'ByteString's.
+--
+-- Note that the wrapped handle is /not/ closed when it receives
+-- end-of-stream; you can use
+-- 'System.IO.Streams.Combinators.atEndOfOutput' to close the handle
+-- if you would like this behaviour.
+--
+-- /Note/: to force the 'Handle' to be flushed, you can write a null string to
+-- the returned 'OutputStream':
+--
+-- > Streams.write (Just "") os
```

Comment:
Add "Since" entry to docstring, i.e.:

```haskell
-- /Since: w.x.y.z/
```

Also please bump minor version and add changelog entry (next version should be 1.3.3.0)